contrib/mhchem/mhchem-manual.html

```diff
@@ -0,0 +1,279 @@
+:::: {(+)}_589{-}[Co(en)3]Cl3
+
+## Bonds
+:::: C6H5-CHO
+:::: A-B=C#D
+mhchem tries to differentiate whether `\ce{-}` should be a bond, a charge or a hyphen.
+:::: A\bond{-}B\bond{=}C\bond{#}D
+:::: A\bond{1}B\bond{2}C\bond{3}D
+:::: A\bond{~}B\bond{~-}C
+:::: A\bond{~--}B\bond{~=}C\bond{-~-}D
+:::: A\bond{...}B\bond{....}C
+:::: A\bond{->}B\bond{<-}C
+
+## Addition Compounds
+:::: KCr(SO4)2*12H2O
+:::: KCr(SO4)2.12H2O
+:::: KCr(SO4)2 * 12 H2O
+
+## Oxidation States
+:::: Fe^{II}Fe^{III}2O4
+
+## Unpaired Electrons, Radical Dots
+:::: OCO^{.-}
+:::: NO^{(2.)-}
+
+## Kr&ouml;ger-Vink Notation <small>(not supported by LaTeX/mhchem yet)</small>
+:::: Li^x_{Li,1-2x}Mg^._{Li,x}$V$'_{Li,x}Cl^x_{Cl}
+:::: O''_{i,x}
+:::: M^{..}_i
+:::: $V$^{4'}_{Ti}
+:::: V_{V,1}C_{C,0.8}$V$_{C,0.2} % upright V = Vanadium, italic V = vacancy
+
+## Equation Operators
+:::: A + B
+:::: A - B % not to be confused with bonds
+:::: A = B % not to be confused with bonds
+:::: A \pm B
+
+## Precipitate and Gas
+:::: SO4^2- + Ba^2+ -> BaSO4 v
+:::: A v B (v) -> B ^ B (^)
+
+## Other Symbols and Shortcuts <small>(not fully supported by LaTeX/mhchem yet)</small>
+:::: NO^* % Excited state
+:::: 1s^2-N % Orbitals
+:::: n-Pr
+:::: iPr
+:::: \ca Fe
+:::: A, B, C; F % Punctuation
+:::: {and others}
+
+## Complex Examples
+:::: Zn^2+  <=>[+ 2OH-][+ 2H+]  $\underset{\text{amphoteres Hydroxid}}{\ce{Zn(OH)2 v}}$  <=>[+ 2OH-][+ 2H+]  $\underset{\text{Hydroxozikat}}{\ce{[Zn(OH)4]^2-}}$
+:::: $K = \frac{[\ce{Hg^2+}][\ce{Hg}]}{[\ce{Hg2^2+}]}$
+:::: $K = \ce{\frac{[Hg^2+][Hg]}{[Hg2^2+]}}$
+:::: Hg^2+ ->[I-]  $\underset{\mathrm{red}}{\ce{HgI2}}$  ->[I-]  $\underset{\mathrm{red}}{\ce{[Hg^{II}I4]^2-}}$
+
+<a name="pu"></a>
+## Physical Units (pu) <small>(KaTeX/MathJax only, not for LaTeX/mhchem)</small>
+::::pu 123 kJ
+::::pu 123 mm2
+There are two conventions regarding the multiplication within units.
+::::pu 123 J s
+::::pu 123 J*s
+There are four conventions regarding divisions.
+::::pu 123 kJ/mol
+::::pu 123 kJ//mol
+::::pu 123 kJ mol-1
+::::pu 123 kJ*mol-1
+There are four main conventions for writing numbers in scientific notation.
+::::pu 1.2e3 kJ
+::::pu 1,2e3 kJ
+::::pu 1.2E3 kJ
+::::pu 1,2E3 kJ
+
+If you need more control than is offered here, take a look at the siunitx extension.
+
+## Compatibility with __LaTeX/mhchem__
+Most of these examples work identically for KaTeX/mhchem and LaTeX/mhchem. Exceptions are indicated. Also, some edge-cases may render differently. I will try to minimize the differences in the future. But even then, KaTeX/mhchem will always be more tolerant of sloppy input (e.g. a missing space) than LaTeX/mhchem.
+
+## Contact and Support
+If you have a question and cannot find an answer---neither here nor with a web search---then
+- mail me (:myaddress:) or
+- create an GitHub issue (https://github.com/mhchem/MathJax-mhchem/issues) or
+- ask at StackExchange (http://meta.chemistry.stackexchange.com/) with the mhchem tag.
```

Comment:
I think we should include KaTeX issue tracker.

Reply:
I've deleted this file entirely. I don't want to maintain future versions of it. Instead, I've added a link to the original mhchem Manual. Hopefully, @mhchem will merge a [PR](https://github.com/mhchem/MathJax-mhchem/pull/8) in which I've written KaTeX into the file.